tests/BaseTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Tests;
+
+use Dotenv\Dotenv;
+use PHPUnit\Framework\TestCase;
+
+class BaseTest extends TestCase
+{
+    protected string $host;
+    protected string $apiKey;
+    protected string $nodeUri;
+    protected string $storeId;
+
+    public static function setUpBeforeClass(): void
+    {
+        $dotenv = Dotenv::createImmutable(__DIR__);
+        $dotenv->safeLoad();
+
+        if (!isset($_ENV['BTCPAY_API_KEY']) || !isset($_ENV['BTCPAY_HOST']) || !isset($_ENV['BTCPAY_STORE_ID']) || !isset($_ENV['BTCPAY_NODE_URI'])) {
```

Comment:
minor nitpick because I found it out not so long ago, you can pass unlimited number of arguments to empty(), so no need to combine separate checks with || 🤯 

https://www.php.net/manual/en/function.isset.php

Reply:
Nice tip, fixed!